src/main/kotlin/no/nav/familie/ef/sak/api/oppgave/OppgaveController.kt

```diff
@@ -41,6 +43,9 @@ class OppgaveController(private val oppgaveService: OppgaveService,
     fun fordelOppgave(@PathVariable(name = "gsakOppgaveId") gsakOppgaveId: Long,
                       @RequestParam("saksbehandler") saksbehandler: String): Ressurs<Long> {
         tilgangService.validerHarSaksbehandlerrolle()
+        if (!tilgangService.validerSaksbehandler(saksbehandler)) {
```

Comment:
Blir det riktig å sjekke om requestparameterverdien fra frontend er lik den som man henter fra SikkerhetsContext ?

Reply:
Både óg:
- På sikt trenger vi dette endepunktet for å fordele oppgaver til _andre_ saksbehandlere. 
- Nå trenger vi å støtte å plukke oppgaven selv - ergo er min tanke at et nytt endepunkt som _ikke_ tar inn saksbehandler kan funke - hvor vi plukker ut saksbehandler fra context. 

`/{gsakOppgaveId}/fordel/innloggetSaksbehandler`  eller noe slikt... 

Men dette her funker jo for behovet vi har nå.

Reply:
Det er også en mulighet å sette default saksbehandler = `SikkerhetContext.hentSaksbehandler()` hvis den ikke er satt, som alternativ til /innloggetSaksbehandler